eli5/formatters/html.py

```diff
@@ -27,13 +28,15 @@
     remaining_weight_color=lambda ws, w_range, pos_neg:
         format_hsl(remaining_weight_color_hsl(ws, w_range, pos_neg)),
     format_feature=lambda f, w, hl: _format_feature(f, w, hl_spaces=hl),
+    format_value=format_value,
     format_decision_tree=lambda tree: _format_decision_tree(tree),
 ))
 
 
 def format_as_html(explanation, include_styles=True, force_weights=True,
                    show=fields.ALL, preserve_density=None,
-                   highlight_spaces=None, horizontal_layout=True):
+                   highlight_spaces=None, horizontal_layout=True,
+                   show_feature_values=True):
```

Comment:
I made it False by default, as before, and added explanations for weight and contribution columns here: https://github.com/TeamHG-Memex/eli5/pull/130/commits/f938d762c9ec3b45f3e151e57679b152f6cbe635#diff-1f22eee3046204ba9c246464571ff48bR108, and this is how it looks like:
![2017-01-09 12 18 56](https://cloud.githubusercontent.com/assets/424613/21761746/21bbf8c8-d666-11e6-9624-0886837c26f0.png)
@kmike do you have suggestions about the text in tooltips?